extension/js/common/platform/store/contact-store.ts

```diff
@@ -297,7 +297,7 @@ export class ContactStore extends AbstractStore {
             cursor.continue();
           }
         }
-      });
+      }, reject);
```

Comment:
I think better just stop using Catch.try in this situation. Instead put our own try/catch block inside the callback method here. Something like

```ts
search.onsuccess = () => {
  try {
    ...
  } catch(e) {
    reject(e)
  }
}
```

Reply:
Then Catch.try can stay as is.

Good find btw.